akka-docs/src/main/paradox/persistence.md

```diff
@@ -32,6 +32,17 @@ changes to these actors from which they can rebuild internal state. This can be
 or starting from a snapshot which can dramatically reduce recovery times. Akka persistence also provides point-to-point
 communication with at-least-once message delivery semantics.
 
+@@@ note
+
+The General Data Protection Regulation (GDPR) require that personal information must be deleted at the request of users.
```

Comment:
Just noticed a small typo: require → requires

Reply:
thx, I'll update and then merge